frontend/test/metabase/scenarios/question/reproductions/27462-no-field-options-for-double-aggregations.cy.spec.js

```diff
@@ -0,0 +1,43 @@
+import { visitQuestionAdhoc, restore, popover } from "__support__/e2e/helpers";
+
+import { SAMPLE_DB_ID } from "__support__/e2e/cypress_data";
+import { SAMPLE_DATABASE } from "__support__/e2e/cypress_sample_database";
+
+const { PRODUCTS_ID, PRODUCTS } = SAMPLE_DATABASE;
+
+describe("issue 27462", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should be able to select field when double aggregating metabase#27462", () => {
+    const questionDetails = {
+      dataset_query: {
+        database: SAMPLE_DB_ID,
+        type: "query",
+        query: {
+          "source-table": PRODUCTS_ID,
+          aggregation: [["count"]],
+          breakout: [["field", PRODUCTS.CATEGORY, null]],
+        },
+      },
+      display: "table",
+      visualization_settings: {},
+    };
+
+    visitQuestionAdhoc(questionDetails, { mode: "notebook" });
+
+    cy.findByRole("button", { name: "Summarize" }).click();
```

Comment:
Follow Kent's advice to use `*ByRole` most of the time if possible.

Reply:
`cy.button()` abstraction does exactly this.

Reply:
@nemanjaglumac Ah nice! Thanks for letting me know that. I actually saw we used `cy.button` in a couple of places, but totally forget about that this time.

A few more questions about that. Let's say we want to create a new test helper.
1. What's a good indicator that that function should exist as a helper? e.g. it's used in multiple places and has certain criteria, otherwise, we could end up with a lot of helpers that are rarely used.
2. If it should be a helper should we use `Cypress.Commands.add` or create them in `__support__/e2e/helpers` and import them directly? And do both of them have their place or should we favor one over another and why?

Reply:
Thank you. This is really helpful.